clients/gtest/gemm_gtest.cpp

```diff
@@ -44,6 +44,8 @@ Representative sampling is sufficient, endless brute-force sampling is not neces
 const
 vector<vector<int>> matrix_size_range = {
                                         {-1, -1, -1, -1, 1, 1},
+                                        { 3, 33,  3,  3, 3, 3},
```

Comment:
{3, 33,3 ,xxxx} valid for NN, as K is reduction dim but not valid for NT. but should be filtered by argument checker. This size is required by TRSM